create_temp_file.py

```diff
@@ -0,0 +1,13 @@
+import os
+import tempfile
+
+for name in ["COM6", "foo"]:
```

Comment:
https://docs.microsoft.com/en-us/windows/win32/fileio/naming-a-file#file-and-directory-names says:

> Do not use the following reserved names for the name of a file:

> CON, PRN, AUX, NUL, COM1, COM2, COM3, COM4, COM5, COM6, COM7, COM8, COM9, LPT1, LPT2, LPT3, LPT4, LPT5, LPT6, LPT7, LPT8, and LPT9. Also avoid these names followed immediately by an extension; for example, NUL.txt is not recommended. For more information, see Namespaces.